docs/reference/operator_parameters.md

```diff
@@ -212,6 +212,11 @@ configuration they are grouped under the `kubernetes` key.
   class](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
   that should be assigned to the Postgres pods. The priority class itself must be defined in advance.
   Default is empty (use the default priority class).
+  
+ * **master_pod_move_timeout**
+   The period of time to wait for the success of Patroni switchovers from master pods on an unschedulable node
```

Comment:
Please make it more explicit, that it includes not only switchover, but the whole pod migration procedure.